### ISSUE TITLE
Home Assistant Add-on

### ISSUE THREAD

**Issue by dannymate** [netbirdio/netbird#497]
Trying to use one of the community base images (I've tried alpine, debian, ubuntu) along with the install from binary option to create an add-on for home assistant for the client. 

However when I get to the point of trying to start the service it gives me 'exit status 1':
netbird service install
netbird service start

I'm wondering if there's any dependencies I need to get it running of if there's some special configuration I've got to do to get NetBird running inside a docker container.

**Comment by mlsmaycon** [comment]
Hello @dannymate sorry to hear that, just so I understand correctly, you are getting the error after installing the client on docker and running the service install and start commands?

If so, we have a official image with a distroless that you can reference netbirdio/netbird:latest  image. If you want to build a custom image, you can see a Dockerfile reference at https://github.com/netbirdio/netbird/blob/main/client/Dockerfile mainly the log file should point to console:

```
ENV NB_LOG_FILE=console
# or ENV WT_LOG_FILE=console
```
and the entrypoint: 
```
ENTRYPOINT [ "/go/bin/netbird","up"]
```

**Comment by dannymate** [comment]
Yes specifically the error occurs after running the netbird service start command.

I'll take a look again with this other image and see what I can do.

**Comment by dannymate** [comment]
Alright I've managed to get it working. I'm not 100% sure what fixed it but I think because I was following the https://netbird.io/docs/getting-started/installation#binary-install guide and I was putting netbird in /usr/bin/netbird. After looking at the dockerfile you sent I switched the path to /go/bin/netbird instead and it seems to like that.

I've got the add-on working anyway can connect to HASSIO through netbird. I'm going to polish it up and see if I can submit it to the community app store.
